apple-pie/styles/style.css

```diff
@@ -0,0 +1,51 @@
+body {
+    width: 800px;
+    margin: 50px;
```

Comment:
Since you are adding a margin to the whole body, you didn't need to add it on the other classes, overall good job!